client/src/partials/patients/documents/views/documentView.js

```diff
@@ -0,0 +1,44 @@
+angular.module('bhima.controllers')
+.controller('DocumentViewController', DocumentViewController);
+
+DocumentViewController.$inject = [
+  'ModalService', '$state', 'DocumentService', 'Upload', 'NotifyService'
+];
+
+function DocumentViewController(Modal, $state, Document, Upload, Notify) {
+  var vm = this;
+
+  /** global objects */
+  vm.patientUuid = $state.params.patient_uuid;
+
+  /** expose to the view */
+  vm.addDocument = addDocument;
+  vm.deleteDocument = deleteDocument;
+
+  /** function add documents modal */
+  function addDocument() {
+    Modal.openAddDocument({ patient_uuid: vm.patientUuid })
+    .then(listDocument);
+  }
+
+  /** delete document */
+  function deleteDocument(uuid) {
+    Document.remove(vm.patientUuid, uuid)
+    .then(function () {
+      Notify.success('FORM.INFOS.DELETE_SUCCESS');
+      listDocument();
+    });
```

Comment:
A good error handling practice is to pass all HTTP promises to `Notify.handleError`.  This ensures that if the server goes offline, or another generic application error occurs, the user is notified.